### PR TITLE
backend/handlers: use same account json

### PR DESCRIPTION
**Pull request by benma**
The two structs were too similar. Makes typing in the frontend easier.

### CODE REVIEW

**Comment by benma** [comment]
`make weblint`:

```

Oops! Something went wrong! :(

ESLint: 7.10.0

TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at Object.keys (/home/marko/go/src/github.com/digitalbitbox/bitbox-wallet-app/frontends/web/node_modules/flat-cache/cache.js:52:19)
    at removeNotFoundFiles (/home/marko/go/src/github.com/digitalbitbox/bitbox-wallet-app/frontends/web/node_modules/file-entry-cache/cache.js:18:35)
    at Object.create (/home/marko/go/src/github.com/digitalbitbox/bitbox-wallet-app/frontends/web/node_modules/file-entry-cache/cache.js:31:5)
    at new LintResultCache (/home/marko/go/src/github.com/digitalbitbox/bitbox-wallet-app/frontends/web/node_modules/eslint/lib/cli-engine/lint-result-cache.js:57:46)
    at new CLIEngine (/home/marko/go/src/github.com/digitalbitbox/bitbox-wallet-app/frontends/web/node_modules/eslint/lib/cli-engine/cli-engine.js:573:29)
    at new ESLint (/home/marko/go/src/github.com/digitalbitbox/bitbox-wallet-app/frontends/web/node_modules/eslint/lib/eslint/eslint.js:423:27)
    at Object.execute (/home/marko/go/src/github.com/digitalbitbox/bitbox-wallet-app/frontends/web/node_modules/eslint/lib/cli.js:285:24)
    at main (/home/marko/go/src/github.com/digitalbitbox/bitbox-wallet-app/frontends/web/node_modules/eslint/bin/eslint.js:142:52)
    at Object.<anonymous> (/home/marko/go/src/github.com/digitalbitbox/bitbox-wallet-app/frontends/web/node_modules/eslint/bin/eslint.js:146:2)
```

@thisconnect is the staging branch bricked? :joy:

**Comment by thisconnect** [comment]
yep I merged a mergeconflict 😪  will open a PR

**Comment by thisconnect** [comment]
@benma fixed in https://github.com/digitalbitbox/bitbox-wallet-app/pull/1032 sorry

**Review by thisconnect — APPROVED** [review]
LGTM

File: frontends/web/src/routes/bitboxbase/bitboxbase.tsx [hunk 1]
```diff
@@ -238,11 +238,7 @@ class BitBoxBase extends Component<Props, State> {
 
         // Only create a new websocket if the bitboxBaseID changed.
         if (this.props.bitboxBaseID !== this.state.bitboxBaseID && this.props.bitboxBaseID) {
-<<<<<<< HEAD
             this.setState({ bitboxBaseID : this.props.bitboxBaseID });
```
Comment (thisconnect) [inline frontends/web/src/routes/bitboxbase/bitboxbase.tsx:241]: Note: one rule removed a space before the colon and the other added a space before the closing bracket, change
`this.setState({ bitboxBaseID : this.props.bitboxBaseID });` to
`this.setState({ bitboxBaseID: this.props.bitboxBaseID });`

**Review by thisconnect — CHANGES_REQUESTED** [review]

File: frontends/web/src/routes/bitboxbase/bitboxbase.tsx (hunk 1 above)
Comment (benma) [reply frontends/web/src/routes/bitboxbase/bitboxbase.tsx]: rebased

**Comment by benma** [comment]
@thisconnect feel free to merge this

**Review by thisconnect — APPROVED** [review]
LGTM

**Pull request merged**